lsp-completion.el

```diff
@@ -501,6 +502,14 @@ Others: TRIGGER-CHARS"
                           (lsp:completion-item-additional-text-edits? resolved-item)))
                cleanup-fn))))
 
+        (when command?
+          (lsp-execute-command
```

Comment:
let's use `lsp-execute-code-action`. We should drop `lsp-execute-command` since it does not work fine when there are multiple workspaces.

Reply:
I mean `lsp--execute-command`

Reply:
Okay.
I think `lsp--execute-command` will also suffered from multiple workspaces.
Since it will only getting command from first action handler in `lsp--find-action-handler`.
Just asking but we will not care about `lsp-execute-command` anymore (thus making generic `lsp-executable-command` become obsoleted) is that correct? Since I saw that `rls` still registering its action handlers via `lsp-execute-command` and the `:action-handlers` seems to point to non-existed function.

Reply:
> I think lsp--execute-command will also suffered from multiple workspaces.

It won't, once we do it right.

1. Local command execution: The command ids are unique(mirroring the VScode internal API). Furthermore, in tramp/docker scenarios you can register one server with different server id's

2. Remote command execution: we should check executeCommandProvider.commands when performing requests to the server. So by default `lsp--find-workspaces-for` should use that to determine which server to pick for `lsp--execute-command` in case there is no local handler.

Reply:
Okay